vm/gvisor/gvisor.go

```diff
@@ -272,7 +272,7 @@ func (inst *instance) Forward(port int) (string, error) {
 		return "", fmt.Errorf("forward port is already setup")
 	}
 	inst.port = port
-	return "stdin", nil
+	return "stdin:0", nil
```

Comment:
Now it fails with this error:
2024/06/25 20:48:35 vm-0: crash: SYZFAIL: failed to parse manager port

Reply:
https://github.com/google/syzkaller/pull/4929